instrumentation/jmx-metrics/library/src/main/java/io/opentelemetry/instrumentation/jmx/yaml/JmxRule.java

```diff
@@ -41,24 +41,22 @@ public class JmxRule extends MetricStructure {
   //     ATTRIBUTE3:
   //       METRIC_FIELDS3
   // The parser never calls setters for these fields with null arguments
-  @Nullable private String bean;
+
   private List<String> beans;
   @Nullable private String prefix;
   private Map<String, Metric> mapping;
 
-  @Nullable
-  public String getBean() {
-    return bean;
-  }
-
-  public void setBean(String bean) {
-    this.bean = validateBean(bean);
-  }
-
   public List<String> getBeans() {
     return beans;
   }
 
+  public void addBean(String bean) {
+    if (beans == null) {
+      beans = new ArrayList<>();
+    }
+    beans.add(validateBean(bean));
+  }
```

Comment:
[for reviewer] simplify a bit by allowing to add any number of beans using either `bean`  (single) or `beans` (list)